src/Components/Agenda/AgendaItemMaintenancePane/AgendaItemMaintenancePane.jsx

```diff
@@ -406,6 +406,7 @@ const AgendaItemMaintenancePane = (props) => {
             <button
               className="usa-button-secondary print-btn"
               onClick={() => togglePrintView(true)}
+              disabled={isCreate}
```

Comment:
Could this be `!readMode` ?

Reply:
you know that's what i had before lol good catch i think that would be better
```suggestion
              disabled={!readMode}
```